packages/eslint-plugin/src/configs/react-base.ts

```diff
@@ -90,7 +89,9 @@ export default {
     'no-template-curly-in-string': 'warn',
     'no-this-before-super': 'warn',
     'no-throw-literal': 'warn',
-    'no-restricted-globals': ['error'].concat(restrictedGlobals),
+    'no-restricted-globals': ['error'].concat(
+      require('confusing-browser-globals')
```

Comment:
Because we point to source, the transpile here can lead to errors if we use the default import (due to ESM interopt). Safer to just use `require` since this package is CJS-only.

This is an Nx repo issue only.